ledger-service/http-json/BUILD.bazel

```diff
@@ -115,8 +115,8 @@ da_scala_test(
         ":Account.dar",
         "//docs:quickstart-model.dar",
     ],
-    # See https://github.com/digital-asset/daml/issues/4281
-    flaky = True,
+#    # See https://github.com/digital-asset/daml/issues/4281
+#    flaky = True,
```

Comment:
If it's no longer flaky, just remove it. `git` won't forget. :wink:

Reply:
don't jinx :)

Reply:
Fixed by https://github.com/digital-asset/daml/pull/4283/commits/5be838a1e0220e94e79adebd3dd70ab7a8983c87